setup.py

```diff
@@ -1,13 +1,16 @@
-import re
+import os
 from setuptools import setup
 
 description = 'Version-bump your software with a single command!'
 
-long_description = re.sub(
-  "\`(.*)\<#.*\>\`\_",
-  r"\1",
-  str(open('README.md', 'rb').read()).replace(description, '')
-)
+# Import the README and use it as the long-description.
+# Note: this will only work if 'README.md' is present in your MANIFEST.in file!
+here = os.path.abspath(os.path.dirname(__file__))
+try:
+    with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
+        long_description = '\n' + f.read()
```

Comment:
Why start with a newline?

Reply:
It just looks nicer in the `*.dist-info/METADATA` file in the wheel package.